README.md

```diff
@@ -16,6 +16,21 @@ composer require libero/content-negotation-bundle
 
 If you're not using [Symfony Flex](https://symfony.com/doc/current/setup/flex.html), you'll need to enable the bundle in your application.
 
+### Route-level negotiation
+
+You can add negotiation at the route level by adding requirements for `_format` and/or `_locale`.
+
+For example, to add an requirement of XML or JSON and English or French to a route:
+
+```yaml
+my_route:
+    path: /path/to/my/page
+    controller: App\Controller\PageController
+    requirements:
+        _format: xml|json
```

Comment:
strange format, compared to a YAML list `['xml', 'json']`?

Reply:
It's regex (see https://symfony.com/doc/current/routing.html#advanced-routing-example).

Reply:
Though we do rely on https://github.com/libero/content-negotiation-bundle/blob/a6e0d89d5e68c30d3dfd01f2b90f0df86966e42e/src/EventListener/RouteLocaleListener.php#L47

Reply:
Maybe should ignore it if the pattern doesn't match.

Reply:
Not sure but I agree the linked code does not allow a generic regex, the `README` should make clear what can be used in general aside from its example

Reply:
Added to the README. We can always revisit.